client/js/components/day/employee.js

```diff
@@ -17,10 +17,16 @@ var Employee = React.createClass({
 
   // connectDragSource added to props by DragSource
   render: function(){
+    var ratings = ["#F26350", "#F26350", "#6BCFFF", "#6BCFFF", "#6BCFFF", "#A8E5A7"];
```

Comment:
Cool so if the ratings are 0 to 5 then id love to make this a object called ratings mapping 0-5 to the hex colors. And then if we have a comment saying what 0 represents and what 5 represents. At the moment is just a little cryptic: I dont know what a 0 rating means or what a 5 rating means and i dont know what any of those hex colors are